wandb/integration/tensorboard/log.py

```diff
@@ -83,7 +83,10 @@ def tf_summary_to_dict(tf_summary_str_or_pb, namespace=""):  # noqa: C901
         return None
 
     def encode_images(img_strs, value):
-        from PIL import Image
+        Image = wandb.util.get_module(
```

Comment:
@raubitsj should weigh in, but I think here we probably just want to print a warning and skip the rest of the function if the module is missing. That way we're still logging scalars at least.

Reply:
Yes,  i think it should be a termwarn(),  make sure to use the feature where it doesnt spam the terminal with the same message,